src/Orbit/CommandLineUtils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef _ORBIT_MAIN_H__
```

Comment:
It seems that the include guard should be the name of the module followed by the name of the file. So maybe `ORBIT_COMMAND_LINE_UTILS_H_` here.

Reply:
Thanks, I've missed this after renaming.